test/javascripts/acceptance/jira-post-menu-test.js

```diff
@@ -270,5 +290,15 @@ acceptance("Jira - post menu", function (needs) {
         `<p><a href="https://jira.example.com/browse/TEST-123">TEST-123</a></p>`
       )
     );
+
+    assert.ok(capturedFields, "The fields payload was captured");
```

Comment:
we'll be linting against non-strict assertions in plugins at some point (we already do in core)

```suggestion
    assert.strictEqual(typeof capturedFields, "object", "The fields payload was captured");
```